lib/site_settings/type_supervisor.rb

```diff
@@ -269,6 +269,8 @@ def validator_for(type_name)
       RegexSettingValidator
     when self.class.types[:string], self.class.types[:list], self.class.types[:enum]
       StringSettingValidator
+    when self.class.types[:host_list]
+      HostListSettingValidator
```

Comment:
I've unfortunately spent too much time and been unable to test this two lines in both `type_supervisor_spec.rb` and `site_setting_extension_spec.rb`. The files have some heavy mocking and cannot reach this a few methods in this file.

Reply:
I think in this case that is OK 👍